scanapp/admin.py

```diff
@@ -24,15 +24,10 @@
 from django.contrib import admin
 
 # Import models from scanapp.models
-from scanapp.models import Scan
-from scanapp.models import ScannedFile
-from scanapp.models import License
-from scanapp.models import Copyright
-from scanapp.models import CopyrightHolder
-from scanapp.models import CopyrightStatement
-from scanapp.models import CopyrightAuthor
-from scanapp.models import Package
-from scanapp.models import ScanError
+from scanapp.models import (
+    Copyright, CopyrightAuthor, CopyrightHolder, CopyrightStatement, License,
+    Package, Scan, ScanError, ScannedFile
+)
```

Comment:
I am sorry but I do not like this style of imports at all, can you use the same style we use elsewhere?